web/thesauruses/c/io.json

```diff
@@ -0,0 +1,97 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "io"
+    },
+    "concepts": {
+        "write_line": {
+            "code": "printf(“Format Specifier”, var1, var2, …., varn);",
+            "name": "Write a line to the console"
+        },
+        "write_line_with_new_line" : {
+            "code": "printf(“Format Specifier”, var1, var2, …., varn);",
+            "name": "Write a line to the console with a return character"  
+        },
+        "read_line": {
+        "code": "scanf(“Format Specifier”, &var1, &var2, …., &varn);",
+        "name": "Read a line of user input from the console"
+        },
+        "read_char": {
+            "code": "getchar()",
+            "name": "Read a single char of user input from the console"
+            },
+        "clear_console_output": {
+            "code": "system(cls)",
+            "comment": "system() is defined in stdlib.h and cls should be in double quotes",
+            "name": "Clear the console output"
+        },
+        "change_console_background_color": {
+            "code" : "unknown",
+            "name": "Change the background color of the console"
+        },
```

Comment:
Are the "unknowns" here and below because you don't know what they are? If that's the case, you could either find out, or just leave `"change_console_background_color"` empty with no code line or anything in it so it's there, but the system knows it doesn't exist (rather than it thinking "unknown" is the way to write it). Same for text color or others below too.